crates/yuck/src/config/script_var_definition.rs

```diff
@@ -71,10 +73,17 @@ impl FromAstElementContent for PollScriptVar {
             let initial_value = Some(attrs.primitive_optional("initial")?.unwrap_or_else(|| DynVal::from_string(String::new())));
             let interval = attrs.primitive_required::<DynVal, _>("interval")?.as_duration()?;
             let (script_span, script) = iter.expect_literal()?;
+
+            let trigger_expr =
+                attrs.ast_optional::<SimplExpr>("trigger")?.unwrap_or_else(|| SimplExpr::Literal(DynVal::from(true)));
```

Comment:
I really dislike the name `trigger`, as that makes it sound like it starts, but never stops the poll-var. The attribute would be better described by a name such as `only-when` or `while`.

Reply:
How about 3 letter `:run [true|false]`?

Reply:
or `:run-while` to be more explicit?

Reply:
I like `:run-while`! the fields in the code should then also be renamed respectively.